src/UserInformation.ts

```diff
@@ -33,6 +37,39 @@ export class Interests {
     this.currencyNetwork = params.currencyNetwork
   }
 
+  public async getUserEarnedMediationFees(
+    networkAddress: string,
+    options: {
+      timeWindowOption?: { startTime: number; endTime: number }
+      decimalsOptions?: DecimalsOptions
+    } = {}
+  ): Promise<EarnedMediationFeesListObject> {
+    const baseUrl = `networks/${networkAddress}/users/${await this.user.getAddress()}/mediation-fees`
```

Comment:
I would put this await this.user.getAddress() on a separate line. I know it is annoying to type that and to come up with yet another variable, but readability comes first. 
Right now just by flying over this code one can miss that we have an async call on this line since the await is hidden inside of the template literal.